clusterloader2/util/unmarshaling.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
I don't think this should live in util/unmarshaling.go

Maybe something like:
config/codec.go